warp10/src/main/java/io/warp10/continuum/sensision/SensisionConstants.java

```diff
@@ -869,6 +869,11 @@ public class SensisionConstants {
    */
   public static final String SENSISION_CLASS_WARPSCRIPT_JVM_FREEMEMORY = "warp.script.jvm.freememory";
 
+  /**
+   * Saturation of FoundationDB network threads
+   */
+  public static final String SENSISION_CLASS_WARPSCRIPT_FDB_BUSYNESS = "warp.script.fdb.busyness";
```

Comment:
Please use `fdb.client.busyness` as the metric name and change the constant name to `SENSISION_CLASS_FDB_CLIENT_BUSYNESS`.

Reply:
Metric name updated